### DISCUSSION TITLE
Timeout error

### DISCUSSION THREAD

**Discussion by hnsantu554** [strongswan/strongswan#2499] [Q&A]
Does strongswan provides error notification on timeout error on responder side?

ERROR_NOTIFY_RETRANSMIT_SEND_TIMEOUT will be sent when initiator reaches the maximum retries on the initiator side

But on the responder side ,After receiving IKE_SA_INIT if IKE_AUTH is not sent by initiator , will strongswan provides error notification on the responder side.

Please help , thank you

**Comment by tobiasbrunner** [comment]
Yes, `ERROR_NOTIFY_HALF_OPEN_TIMEOUT`.

**Reply by hnsantu554** [reply]
Thanks for the update , what is the time taken to occur ERROR_NOTIFY_HALF_OPEN_TIMEOUT? Is there any configuration parameter for the half open connection?

**Reply by tobiasbrunner** [reply]
30 seconds, `charon.half_open_timeout` in [strongswan.conf](https://docs.strongswan.org/docs/5.9/config/strongswanConf.html). Please try to do your own research in the future.
